docs/appendix/changelog.md

```diff
@@ -21,52 +22,48 @@ Check out [this tutorial](/tutorials/projects/filtered-camera/) for more informa
 
 {{% /changelog %}}
 
-{{% changelog color="added" title="Raspberry Pi 5 Support" %}}
+{{% changelog date="2023-12-01" color="added" title="Raspberry Pi 5 Support" %}}
```

Comment:
If possible I think we shouldn't put the first of the month since that's not accurate

Reply:
changed to the end of the month. I don't think any dates except the vision service are accurate. But we did used to announce everything at the end of the month so may as well stick with that